libcodechecker/analyze/analyzers/analyzer_clangsa.py

```diff
@@ -128,15 +128,12 @@ def construct_analyzer_cmd(self, result_handler):
                                          '-Xclang', checker_name])
 
             if config.ctu_dir:
-                env = analyzer_env.get_check_env(config.path_env_extra,
-                                                 config.ld_lib_path_extra)
-                triple_arch = ctu_triple_arch.get_triple_arch(self.buildaction,
-                                                              self.source_file,
-                                                              config, env)
                 analyzer_cmd.extend(['-Xclang', '-analyzer-config',
                                      '-Xclang',
-                                     'xtu-dir=' + os.path.join(config.ctu_dir,
-                                                               triple_arch)])
+                                     'xtu-dir=' + self.get_xtu_dir(),
```

Comment:
We've inherited "XTU" name from somewhere else so we made the decision to call everything we do "CTU". Although the legacy parameter here (which was not changed in clang yet) says XTU, we should rather consequently call these CTU in the code. Someone may change it in clang as well in the future. :)

Reply:
A `FIXME` or a comment would be nice here, that this parameter needs to be changed, or we should create a separate issue.

Reply:
https://github.com/Ericsson/clang/issues/208